docs/api-network.rst

```diff
@@ -9,7 +9,7 @@ The ``network`` package holds classes for interacting with the Ethereum blockcha
 ``brownie.network.main``
 ========================
 
-The ``main`` module contains methods for conncting to or disconnecting from the network. All of these methods are available directly from ``brownie.network``.
```

Comment:
The current version is correct. You either `connect to` or `disconnect from` a network. Your version would imply that you `connect from` which doesn't read correctly.

I'd also suggest improving the commit message - https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/

Reply:
Thanks for your feedback, my point was to add the missing 'e' in connecting but unfortunately I have removed 'to' also.
I will update my pull request.